proposals/devcontainer-features.md

```diff
@@ -108,12 +108,14 @@ The `id` format specified dicates how a supporting tool will locate and download
 | Type | Description | Example |
 | :--- | :--- | :--- |
 | `<oci-registry>/<namespace>/<feature>[:<semantic-version>]` | Reference to feature in OCI registry(*) | ghcr.io/user/repo/go <br> ghcr.io/user/repo/go:1 <br> ghcr.io/user/repo/go:latest|
-| `https://<uri-to-feature-tgz>` | Direct HTTPS URI to a tarball. | https://github.com/user/repo/releases/devcontainer-feature-go.tgz |
-| `./<path-to-feature-dir>`| A relative directory to folder containing a devcontainer-feature.json. | ./myGoFeature |
+| `https://<..uri..>/devcontainer-feature-<feature>.tgz` | Direct HTTPS URI to a tarball. | https://github.com/user/repo/releases/devcontainer-feature-go.tgz |
+| `./<path-to-feature-dir>`| A relative directory(**) to folder containing a devcontainer-feature.json. | ./myGoFeature |
+
 
-`
 (*) OCI registry must implement the [OCI Artifact Distribution Specification](https://github.com/opencontainers/distribution-spec).  Some implementors can be [found here](https://oras.land/implementors/).
 
+(**) The provided path is relative to the workspace folder root (the `--workspace-folder` ).
```

Comment:
Suggestion: 
(**) The provided path must be inside a directory or subdirectory of the workspace folder root (the `--workspace-folder`).

It can be just me but this sentence sounded a little contradicting to :point-down as the user can have the directory relative to the workspace folder but outside of it. 

> A feature can be referenced directly in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) by relative path _inside_ the project root directory, or a subdirectory of the project root. Note that the project root directory is indicated by the required `--workspace-folder <...>` parameter.  
> A local feature may not be referenced outside of the project directory (`../` is not allowed), nor is an absolute path allowed.

Reply:
Hmm, I mean for them to sound the same, only the workspace-folder and anything inside of that directory.